src/main/client/src/App.module.css

```diff
@@ -1,3 +1,3 @@
-.pageHeader {
-    margin-left: 0.5em;
+.container {
+    margin: 10px;
```

Comment:
Should we be avoiding using pixels? Especially if we want the app to be mobile friendly